frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -130,6 +133,9 @@ export const savedInsightsLogic = kea<savedInsightsLogicType<InsightsResult>>({
         ],
     },
     listeners: ({ actions }) => ({
+        addGraph: ({ type }) => {
+            router.actions.push(`/insights?insight=${type.toString().toUpperCase()}&backToURL=/saved_insights`)
```

Comment:
`/saved_insights` will no longer exist, right?

Reply:
`/saved_insights` should still exist as it is its own menu nav tab now